# The states used in vehicle control
# x, y, heading and speed
float64[] x
float64[] y
float64[] psi
float64[] v